spec/forms/respondent_form_spec.rb

```diff
@@ -150,7 +155,7 @@
       describe 'when and ACAS number is given' do
         before { respondent_form.has_acas_number = 'true' }
 
-        it     { expect(respondent_form).not_to validate_presence_of(:no_acas_number_reason) }
```

Comment:
why is it not important to prove that we are not validating no_acas_number_reason in this scenario ?

Reply:
would this be a better alternative? The test cases pass with this change:

```
describe 'when an ACAS number is given' do
  before { respondent_form.has_acas_number = 'true' }

  it     { expect(respondent_form).to validate_presence_of(:acas_early_conciliation_certificate_number) }
  it     { expect(respondent_form).to_not validate_presence_of(:no_acas_number_reason) }
end

describe 'when an ACAS number is given' do
  before { respondent_form.has_acas_number = 'false' }

  it     { expect(respondent_form).to validate_presence_of(:no_acas_number_reason) }
  it     { expect(respondent_form).to_not validate_presence_of(:acas_early_conciliation_certificate_number) }
end
```

Reply:
yes that looks right